tests/installation/bootloader_uefi.pm

```diff
@@ -19,7 +19,10 @@ use utils;
 
 # hint: press shift-f10 trice for highest debug level
 sub run() {
-    my ($self) = @_;
+    if (get_var('DUALBOOT')) {
+        # send ESC to prevent tianocore from booting from hard disk...
+        send_key_until_needlematch('bootloader-shim-import-prompt', 'esc', 10, 5);
```

Comment:
what if there is no shim to import? That's why recently I changed the next `assert_screen` to check for both tags.